categorysite.html

```diff
@@ -66,117 +66,116 @@ <h4>Varukorg</h4>
 			</div>
 		</nav>
 <!-- header end -->
-    <div class="container col-12 col-md-6 flex-column">
-    	<div class="row">
+    <div class="container col-sm-12 col-md-12 d-flex flex-nowrap">
+    	<div class="col-md-7">
     	<content class="content">
```

Comment:
För att innehållet, dvs kategorierna skall positioneras korrekt med `col-`, måste alla kategorier omslutas om en row klass.

```suggestion
    	<content class="content row">
```